### PR TITLE
[Instrumentation.Process] Enable nullable

### PR DESCRIPTION
**Pull request by Kielek**
Towards #894

## Changes

Nullable for Process project.

For significant contributions please make sure you have completed the following items:

* ~~[ ] Appropriate `CHANGELOG.md` updated for non-trivial changes~~
* ~~[ ] Design discussion issue #~~

### CODE REVIEW

**Review by TimothyMothra — APPROVED** [review]

**Comment by TimothyMothra** [comment]
@Kielek I would encourage you to create an Issue to track all of these individual PRs. :)

**Comment by Kielek** [comment]
> @Kielek I would encourage you to create an Issue to track all of these individual PRs. :)

Done, both PRs linked to #894

File: src/OpenTelemetry.Instrumentation.Process/MeterProviderBuilderExtensions.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public static class MeterProviderBuilderExtensions
     /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
     public static MeterProviderBuilder AddProcessInstrumentation(
         this MeterProviderBuilder builder,
-        Action<ProcessInstrumentationOptions> configure = null)
+        Action<ProcessInstrumentationOptions>? configure = null)
```
Comment (alanwest) [inline src/OpenTelemetry.Instrumentation.Process/MeterProviderBuilderExtensions.cs:36]: Same story here... https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/893#discussion_r1072846852

Comment (Kielek) [reply src/OpenTelemetry.Instrumentation.Process/MeterProviderBuilderExtensions.cs]: Fixed: 3018230d179924fdfe7d99896d571ce803b1b6f6

**Review by alanwest — APPROVED** [review]

**Pull request merged**